.npmrc

```diff
@@ -0,0 +1 @@
+legacy-peer-deps=true
```

Comment:
@alexander-veit - This will automatically use `--legacy-peer-deps` for all `npm install` commands so we don't need to remember to explicitly use it.